ecommerce-ui/src/App.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {shallow, render, mount} from 'enzyme';
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+
+/*it('shallow renders', () => {
+  const wrapper = shallow(<App />);
+  expect(wrapper).toMatchSnapshot();
+});
```

Comment:
Expect to see some snapshot testing. You can follow the sample code from class here:
https://github.com/modern-web-application-uw19/week4-code-samples/blob/master/snapshot-testing/src/App.test.js
remember that your tests will initially fail, then you can save the snapshot from the terminal dialogue as a file so that tests should pass in the future.